### PR TITLE
4.6 Platform Tile Release Notes

### PR DESCRIPTION
**Pull request by pavankrish123**
[WIP]: @okilker  please review the new installation document

### CODE REVIEW

File: docs-content/installing.html.md.erb [hunk 1]
```diff
@@ -151,10 +151,10 @@ Starting 4.6.* version of the tile, AppDynamics Nozzle is completely scalable to
...
 
 
-## <a id='deploy-multiple-tiles-on-multiple-instances'></a> Deploying Multiple Tiles on Multiple Cloud Foundry Instances to a Single AppDynamics Controller
+## <a id='deploy-multiple-tiles-on-multiple-instances'></a> Monitoring Multiple Cloud Foundry Instances
 
-You may wish to deploy multiple AppDynamics service broker tiles on to multiple Cloud Foundry instances, using a single AppDynamics Controller.
+You may wish to deploy multiple AppDynamics Platform Monitoring tiles on to multiple Cloud Foundry instances, using a single AppDynamics Controller.
 The metrics corresponding to each of the cloud foundry instances are recorded under tier names which defaults to the system domain's value of the Cloud Foundry deployment e.g: `sys.pie-multi-az-blue.cfplatformeng.com`
-In addtion, you can overwrite the tier name with a custom value by filling in the **Nozzle Tier Name** field as mentioned in the section above.
+In addtion, you can also overwrite the tier name with a custom value by filling in the **Nozzle Tier Name** field as mentioned in the section above.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: addition

File: docs-content/installing.html.md.erb [hunk 2]
```diff
@@ -12,9 +12,8 @@ multi-foundation dashboard and overiding default configurations.
 
 ## <a id='overview-appd-nozzle'></a> Overview
 
-AppDynamics Nozzle aggregates and forwards logs and metrics from the **Loggregator [Firehose](https://docs.cloudfoundry.org/loggregator/architecture.html#firehose)**
-to the AppDynamics Controller. The Controller processes the combined data stream of logs, HTTP events, and
-container metrics from all applications, as well as detailed performance metrics on the health of the underlying
+AppDynamics Nozzle aggregates and forwards metrics from the **Loggregator [Firehose](https://docs.cloudfoundry.org/loggregator/architecture.html#firehose)**
+to the AppDynamics Controller. The Controller processes the combined data stream of  detailed performance metrics on the health of the underlying
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: extra space (of  detailed)

File: docs-content/installing.html.md.erb [hunk 3]
```diff
@@ -145,8 +144,17 @@ to report the metric the filter entry will be
 For 4.4.x controller, please create an applicaiton on the controller and create a `Go SDK` tier associated with that application and record the tier-id.
 Enter the AppName, Tier, TierID and Node names in the corresponding fields of the **Overriding Defaults** form.
 
+
+## <a id='Scaling'></a> Scalability
+
+Starting 4.6.* version of the tile, AppDynamics Nozzle is completely scalable to any number of desired instances. However,
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: Starting with the 4.6.*

File: docs-content/installing.html.md.erb [hunk 4]
```diff
@@ -145,8 +144,17 @@ to report the metric the filter entry will be
...
+it is recommended to scale to the number of traffic controller instances. See the [official cloud foundry recommendation](https://docs.cloudfoundry.org/loggregator/log-ops-guide.html#scaling-nozzles)
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: to scale the number of nozzle instances to match the number of traffic controller instances.

**Comment by pavankrish123** [comment]
Thanks @okilker - fixed review comments

File: docs-content/installing.html.md.erb [hunk 5]
```diff
@@ -3,14 +3,24 @@ title: Installing and Configuring AppDynamics Platform Monitoring for PCF
 owner: Partners
 ---
 
-This topic describes how to install and configure AppDynamics Platform Monitoring for Pivotal Cloud Foundry (PCF). The tile allows you to deploy a Firehose Nozzle Agent that collects various KPI metrics and
-a Dashboard that automatically creates health-rules and custom dashboards associated with the metrics.
+This topic describes how to install and configure AppDynamics Platform Monitoring for Pivotal Cloud Foundry (PCF).
+The tile allows you to deploy  AppDynamics Firehose Nozzle that collects various KPI metrics and a Dashboard that
+automatically creates health-rules and custom dashboards associated with the metrics.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: health rules (no hyphen)

File: docs-content/installing.html.md.erb [hunk 6]
```diff
@@ -3,14 +3,24 @@ title: Installing and Configuring AppDynamics Platform Monitoring for PCF
...
+
+Additionally the tile takes in advanced configuration to customize metric collection, configuring
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: ~takes in~  allows for

File: docs-content/installing.html.md.erb [hunk 7]
```diff
@@ -33,14 +43,14 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
 1. Double-click the **AppDynamics Platform Monitoring for PCF** tile in the Installation Dashboard.
    A Settings tab is displayed.
 
-1. In the **Settings** tab, select **AppDynamics Nozzle Configuration**.
-![Settings tab for AppDynamics Nozzle Configuration](./images/appd-nozzle-config.png)
+1. Fill in the details for the following forms on the tile.
+
+### <a id='nozzle configuration'></a> AppDynamics Nozzle Configuration
+
+This form take in information to establish connection with Cloud Foundry Loggregator and AppDynamics Controller.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: takes

Comment (okilker) [reply docs-content/installing.html.md.erb]: establish a connection with the Cloud

File: docs-content/installing.html.md.erb [hunk 8]
```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
...
+* (or) Create a new account in the doppler.firehose group with the correct permissions.
+      For more information, see [UAA Client](https://github.com/cf-platform-eng/firehose-nozzle#option-2-uaa-client).
 
-### AppDynamics Nozzle app
+### <a id='dashboard configuration'></a> AppDynamics Dashboard Configuration
 
-1. Open the AppD **Apps Manager** and navigate to the appdynamics space.
-1. In the **appdnozzle`-<version>`** app, open the **Settings** tab.
-1. Click **REVEAL USER PROVIDED ENV VARS**.
+This form takes in information to create Dashboard and associated health rules on the AppDynamics Controller
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: to create a Dashboard

File: docs-content/installing.html.md.erb [hunk 9]
```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
...
 
-1. Add the proxy environment variables and click **SAVE**.
+* **Username for Reporting Custom Dashboard**: Username of an account with custom dashboard creation permissions.
+* **Password**: Password for the Controller account.
 
-    ![AppDynamics Proxy Environment Variables](./images/add-envars-save.png)
+### <a id='Advanced metrics collection'></a> (Optional) Extending Metrics Collection
 
-1. Click the **Restart** button to restart the app.
+Starting 4.6 version of the tile we have included a features to collect additional metrics in addition to
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: Starting with the 4.6.*

File: docs-content/installing.html.md.erb [hunk 10]
```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
...
+the default standard KPI metrics that are collected by AppDynamics Nozzle.
 
-    ![AppDynamics Nozzle Restart button](./images/restart-app-ui.png)
+![Settings tab for AppDynamics Advanced Configuration](./images/3.png)
 
-### AppDynamics KPI Dashboard app
+* **Metrics**: The type of metrics that are to be collected by the Nozzle. AppDynamics Nozzle collects `CounterEvent` and `ValueMetric` metrics to record standard KPI metrics.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: The type of metrics that are collected by the Nozzle.

File: docs-content/installing.html.md.erb [hunk 11]
```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
...
+It is highly recommend to keep these two metric types selcted in the form.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: selected

File: docs-content/installing.html.md.erb [hunk 12]
```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
...
 
-1. Open the AppD **Apps Manager** and navigate to the **appdynamics space**.
-1. In the **appdpcfdashboard`-<version>`** app, open the **Settings** tab.
-1. Click **REVEAL USER PROVIDED ENV VARS**.
+* **Extended Metrics JSON Filter**: A filter is applied to collected metric to limit the data that is processed. This JSON defines
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: A filter is applied to the collected metric

File: docs-content/installing.html.md.erb [hunk 13]
```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
...
+that filter. The filter is a simple dictionary with keys mapping to `origin` tag of the metric and value representing a list of `metric names`
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: mapping to the `origin` tag

File: docs-content/installing.html.md.erb [hunk 14]
```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
...
-You must select a `Go SDK` tier, but the tier id is not necessary for Advanced Configuration.
+* **UAA Token Endpoint**: UAA token end point of the cloudfoundry. This can be found by doing `$ cf curl /v2/info | grep token_endpoint`
+* **Nozzle Application Name**: Enter a name to identify the Nozzle Application on the Controller, such as **_test-app-nozzle_**.
+* **Nozzle Tier Name**: Enter a name for the Nozzle Tier that corresponds to the Nozzle Application Name, such as **_test-app-tier_**. This MUST be a Go SDK Tier.
+* **Nozzle Node Name**: Enter a name for the Nozzle Node that corresponds to the Nozzle Application Name, such as **_test-app-node_**.
+* **Nozzle Tier ID**: Enter the component ID of the GoLang SDK tier.
 
-**To rename the Nozzle Application, Tier and Node names**
+#### <a id='44x Controllers'></a> 4.4.x Controllers
+For 4.4.x controller, please create an applicaiton on the AppDynamics Controller and create a `Go SDK` tier associated with that application and record the tier-id.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: For 4.4.x controller, ~please~ create an application on the AppDynamics Controller. Create a `Go SDK` tier associated with that application and record the tier-id.

File: docs-content/installing.html.md.erb [hunk 15]
```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
...
-   ![AppDynamics Nozzle configuration form](./images/nozzle-adv-config.png)
-    * **Nozzle Application Name**: Enter a name to identify the Nozzle Application on the Controller, such as **_test-app-nozzle_**.
-    * **Nozzle Tier Name**: Enter a name for the Nozzle Tier that corresponds to the Nozzle Application Name, such as **_test-app-tier_**. This MUST be a Go SDK Tier.
-    * **Nozzle Node Name**: Enter a name for the Nozzle Node that corresponds to the Nozzle Application Name, such as **_test-app-node_**.
-    * **Nozzle Tier ID**: Enter the component ID of the GoLang SDK tier.
 
+## <a id='Scaling'></a> AppDynamics Nozzle Scalability
 
-1. In the Installation Dashboard, click **Apply Changes** and wait for the changes to be applied.
+Starting with 4.6.* version of the tile, AppDynamics Nozzle is completely scalable to any number of desired instances. However,
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: Starting with the 4.6.*

File: docs-content/index.html.md.erb [hunk 16]
```diff
@@ -16,19 +23,19 @@ a Dashboard that automatically creates health-rules and custom dashboards associ
     <th>Details</th>
     <tr>
         <td>Version</td>
-        <td>v4.5.605</td>
+        <td>v4.6.18</td>
     </tr>
     <tr>
         <td>Release date</td>
-        <td>February 4, 2019</td>
+        <td>March 6, 2019</td>
```
Comment (okilker) [inline docs-content/index.html.md.erb]: Is March 6th the official release date?

Comment (pavankrish123) [reply docs-content/index.html.md.erb]: Nope. Just a placeholder for now

**Pull request merged**